indico/modules/events/abstracts/templates/management/notification_tpl_form.html

```diff
@@ -1,10 +1,9 @@
 {% from 'forms/_form.html' import form_header, form_footer, form_rows, form_row, form_fieldset %}
 
 {{ form_header(form) }}
-    {{ form_row(form.title) }}
+    {{ form_rows(form, fields=('title', 'stop_on_match')) }}
```

Comment:
That's what I am saying, I don't see a point of having the "E-mail template" fieldset. cc @OmeGak, @ThiefMaster, @pferreir ?